zuul.d/jobs.yaml

```diff
@@ -755,18 +755,55 @@
       - playbooks/ansible-test-base/post.yaml
     required-projects:
       - name: github.com/ansible/ansible
+      - name: github.com/ansible-collections/openvswitch.openvswitch
     timeout: 3600
     vars:
+      ansible_collections_repo: github.com/ansible-collections/openvswitch.openvswitch
       ansible_test_command: network-integration
       ansible_test_integration_targets: "openvswitch_.*"
 
+- job:
+    name: ansible-test-units-base
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+    run: playbooks/ansible-test-base/run.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+    timeout: 3600
+    vars:
+      ansible_test_command: units
+
+
+- job:
+    name: ansible-test-network-unit-openvswitch
+    abstract: true
```

Comment:
remove abstract, we'll use this job as default no need for python specific version.